maps/away/scenarios/enviro_testing_facility/enviro_testing_facility.dm

```diff
@@ -0,0 +1,118 @@
+
+/singleton/submap_archetype/enviro_testing_facility
+	map = "Environmental Testing Facility"
+	descriptor = "Environmental Testing Facility."
+
+// --------------------------------------------------- sector
+
+/obj/effect/overmap/visitable/sector/enviro_testing_facility
+	name = "Anastasia-Archippos, Barren Planet"
+	desc = "\
+		Barren planet, covered in loose rocks, mountains, craters, with a expansive cave system deep underground. \
+		The planet, however, holds a standard breathable atmosphere, and there are some traces of carbon-based life. \
+		Landing site is in a valley, near some facility carved into the mountain.\
+		"
+	icon_state = "globe2"
+	color = "#e6a66a"
+	comms_support = TRUE
+	initial_generic_waypoints = list(
+		"nav_enviro_testing_facility_surface_1a",
+		"nav_enviro_testing_facility_surface_1b",
+		"nav_enviro_testing_facility_surface_1c",
+		"nav_enviro_testing_facility_surface_1d",
+
+		"nav_enviro_testing_facility_surface_2a",
+		"nav_enviro_testing_facility_surface_2b",
+		"nav_enviro_testing_facility_surface_2c",
+		"nav_enviro_testing_facility_surface_2d",
+
+		"nav_enviro_testing_facility_surface_3a",
+		"nav_enviro_testing_facility_surface_3b",
+		"nav_enviro_testing_facility_surface_3c",
+		"nav_enviro_testing_facility_surface_3d",
+
+		"nav_enviro_testing_facility_surface_4a",
+		"nav_enviro_testing_facility_surface_4b",
+		"nav_enviro_testing_facility_surface_4c",
+		"nav_enviro_testing_facility_surface_4d",
+
+		"nav_enviro_testing_facility_surface_5a",
+		"nav_enviro_testing_facility_surface_5b",
+		"nav_enviro_testing_facility_surface_5c",
+		"nav_enviro_testing_facility_surface_5d",
+	)
+
+/obj/effect/overmap/visitable/sector/enviro_testing_facility/generate_ground_survey_result()
+	..()
+	if(prob(50))
+		ground_survey_result += "<br>Evidence of lava tubes being present in the subsurface"
+	if(prob(50))
+		ground_survey_result += "<br>Water ice pockets detected deep underground"
+	if(prob(50))
+		ground_survey_result += "<br>No geothermal activity observed in the planetary core"
+	if(prob(50))
+		ground_survey_result += "<br>Micro-textural analysis indicates availability of fissile material"
+	if(prob(50))
+		ground_survey_result += "<br>Surface soil may provide adequate radiation shielding"
+	if(prob(50))
+		ground_survey_result += "<br>Silicon carbides found deep in the crust"
+	if(prob(50))
+		ground_survey_result += "<br>Oxygen found in locally stable metal oxides"
+	if(prob(40))
+		ground_survey_result += "<br>Regolith rich in heavy silicate alloys"
+	if(prob(30))
+		ground_survey_result += "<br>Relatively high abundance of fusile material, accumulated on the surface by the solar wind"
+	if(prob(10))
+		ground_survey_result += "<br>Traces of fusile material"
+	if(prob(10))
+		ground_survey_result += "<br>Carbon nanotubes naturally found in the regolith"
+
+// --------------------------------------------------- misc
+
+#define NETWORK_ENVIRO_TESTING_FACILITY "Env-Test Facility Zoya"
```

Comment:
Forgot one that it didn't took for some reason (maybe i forgot to save it)

https://github.com/Aurorastation/Aurora.3/blob/master/.github/guides/STYLE.md#undef-any-macros-you-create-unless-they-are-needed-elsewhere

`We do not want macros to leak outside their file, this will create odd dependencies that are based on the filenames. Thus, you should #undef any macro you make.`

`If you want more than one files to use macros, put them in somewhere as a file in __DEFINES. `

Reply:
removed the define